### DISCUSSION TITLE
How to check follow/block/mute status?

### DISCUSSION THREAD

**Discussion by GreenFootballs** [bluesky-social/atproto#1424] [How to do it?] [answered]
Is there a way to use the API to quickly get graph info on a specific account — i.e. whether you’re following or blocking/muting them?

**Comment by dholms** [comment, marked as answer]
If you fetch their profile with `app.bsky.feed.getProfile`, it will include that info in the `viewer` state 👍

**Comment by GreenFootballs** [comment]
Thanks, that works - but I think you meant `app.bsky.actor.getProfile`.
